package.json

```diff
@@ -7,7 +7,7 @@
   },
   "scripts": {
     "format:check": "prettier --check '{''{lib,test}/!(test/**/fixtures)/**/*,*}.{js,ts,json,yml}'",
-    "format": "prettier --write '{''{lib,test}/!(test/**/fixtures)/**/*,*}.{js,ts,json,yml}'",
+    "format": "prettier --write '{''{src,test}/!(fixtures)/**/*,*}.{js,ts,json,yml}'",
```

Comment:
please rebase once the prettier PR is merged. So we don't see this diff

Reply:
Will do

Reply:
still seeing this diff? shouldn't rebase take care of it or something had to change again?

Reply:
Yes, I rebased and the change went away, can't figure it out.